### PR TITLE
Update validation.js to accept waitUntil string or Array

### PR DESCRIPTION
**Pull request by coptere**
This simple patch to match Puppeteer API options. (no verificication on string/valid string in array alternative)
https://github.com/GoogleChrome/puppeteer/blob/v1.8.0/docs/api.md#pagegotourl-options
...waitUntil can be <string|Array<string>>

### CODE REVIEW

**Comment by Jameskmonger** [comment]
@coptere This [build is failing](https://travis-ci.org/alvarcarto/url-to-pdf-api/builds/436534653) due to some linting issues.

**Comment by coptere** [comment]
@Jameskmonger eslint errors fixed.

File: src/core/render-core.js [hunk 1]
```diff
@@ -81,10 +81,12 @@ async function render(_opts = {}) {
       await page.emulateMedia('screen');
     }
 
-    logger.info('Setting cookies..');
-    opts.cookies.map(async (cookie) => {
-      await page.setCookie(cookie);
-    });
+    if (opts.cookies && opts.cookies.length > 0) {
```
Comment (kimmobrunfeldt) [inline src/core/render-core.js]: Hi, thanks for the efforts! This PR seems to also change the cookie logic. Could you separate the cookie change from this PR to another one and explain why it's also needed?

**Comment by kimmobrunfeldt** [comment]
Closing due to inactivity.

**Pull request closed without merging**